geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/AcceptorImpl.java

```diff
@@ -317,6 +319,12 @@
 
   private final ServerConnectionFactory serverConnectionFactory;
 
+  private final Properties nonDefault = new Properties();
+  private final DistributionConfigImpl distributionConfigImpl =
+      new DistributionConfigImpl(nonDefault);
```

Comment:
when does the nonDefaultProperties get populated?  I think I'm misunderstanding how a nonDefault value actually is configured.